ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/SpecConfigBuilder.java

```diff
@@ -225,7 +225,9 @@ private void validate() {
     validateConstant("depositContractAddress", depositContractAddress);
 
     altairBuilder.ifPresent(AltairBuilder::validate);
-    bellatrixBuilder.ifPresent(BellatrixBuilder::validate);
+    bellatrixBuilder
+        .filter(BellatrixBuilder::isBellatrixEnabled)
+        .ifPresent(BellatrixBuilder::validate);
```

Comment:
We only require the Bellatrix options to be present if Bellatrix is actually being enabled. Previously this would happen naturally because if `BELLATRIX_FORK_EPOCH` wasn't set we'd never create the builder, but now that we have a couple of options that can be overridden the bellatrixBuilder in `SpecConfigBuilder` always winds up being created.  Rather than being really careful to avoid creating the builder unnecessarily, it's safer to only apply the validation if we're actually going to use those values.

We also ensure we don't create a `SpecConfigBellatrix` instance when building since we haven't validated the values, thus ensuring nothing can depend on them (as they aren't available).